components/base/BaseDropdown.vue

```diff
@@ -35,6 +49,7 @@ const props = withDefaults(
   }>(),
   {
     flavor: 'button',
+    buttonColor: 'default',
```

Comment:
We also need to have the default value set in https://github.com/shuriken-ui/nuxt/blob/main/app.config.ts